geomesa-kafka/geomesa-kafka-datastore/src/test/scala/org/locationtech/geomesa/kafka/utils/GraphVizFilterVisitor.scala

```diff
@@ -0,0 +1,144 @@
+/***********************************************************************
+* Copyright (c) 2013-2016 Commonwealth Computer Research, Inc.
+* All rights reserved. This program and the accompanying materials
+* are made available under the terms of the Apache License, Version 2.0
+* which accompanies this distribution and is available at
+* http://www.opensource.org/licenses/apache2.0.php.
+*************************************************************************/
+
+package org.locationtech.geomesa.kafka.utils
+
+import org.opengis.filter._
+import org.opengis.filter.spatial._
+import org.opengis.filter.temporal._
+
+import scala.collection.JavaConversions._
+
+class GraphVizFilterVisitor extends FilterVisitor {
```

Comment:
Yeah, we need to find a place to dump esoteric, but somewhat useful filter code...